integration/testrunner/grpcrunner/grpc.go

```diff
@@ -111,21 +113,40 @@ func TestGRPCEndpoint(ctx context.Context, tb testing.TB, g *GRPC) {
 	if g.BigQueryClient == nil {
 		bqClient, err := utils.MakeClient(ctx, g.ProjectID)
 		if err != nil {
-			tb.Errorf("BigQuery request failed: %v", err)
+			t.Errorf("BigQuery request failed: %v", err)
 		}
-		tb.Cleanup(func() {
+		t.Cleanup(func() {
 			if err := bqClient.Close(); err != nil {
-				tb.Errorf("Failed to close the BQ client: %v", err)
+				t.Errorf("Failed to close the BQ client: %v", err)
 			}
 		})
 		g.BigQueryClient = bqClient
 	}
 
-	g.runHelloCheck(ctx, tb)
-	g.runFailCheck(ctx, tb)
-	g.runFibonacciCheck(ctx, tb)
-	g.runAdditionCheck(ctx, tb)
-	g.runFailOnFourCheck(ctx, tb)
+	t.Run("hello_req_unary_success", func(t *testing.T) {
+		t.Parallel()
+		g.runHelloCheck(ctx, t)
```

Comment:
Should we just inline these?

Reply:
Done. I also removed the top `t.Helper()` in this file. Here locates the real testing logic rather than a test helper. I found it's hard to locate the line of error if I'm putting `t.Helper()` because it gave me the line in the `main_test.go`